packages/squiggle-lang/src/ast/peggyHelpers.ts

```diff
@@ -163,7 +193,22 @@ export function nodeProgram(
       symbols[statement.variable.value] = statement;
     }
   }
-  return { kind: "Program", imports, statements, symbols, location };
+  return {
+    kind: "Program",
+    imports: imports.map((imp) => assertKind(imp, "Import")),
+    statements: statements.map(assertStatement),
+    result: result ? assertExpression(result) : null,
```

Comment:
All these assertions are not necessary if Peggy grammar is correct, but we don't have type safety, and the performance hit is negligible.